### PR TITLE
Metricfrequencyprocessor

### PR DESCRIPTION
**Pull request by ppawelecsumo**

### CODE REVIEW

File: pkg/processor/metricfrequencyprocessor/Makefile [hunk 1]
```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```
Comment (sumo-drosiek) [inline pkg/processor/metricfrequencyprocessor/Makefile:1]: newline :see_no_evil:

Comment (ppawelecsumo) [reply pkg/processor/metricfrequencyprocessor/Makefile:1]: This file is copied from other processors. They all omit this newline.

**Comment by pmalek-sumo** [comment]
Please rebase, fix the build issue

```
2021-09-28T11:01:13.252Z	ERROR	cmd/root.go:91	failed to run	{"error": "failed to compile the OpenTelemetry Collector distribution: exit status 2. Output: \"# github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricfrequencyprocessor\\n../../pkg/processor/metricfrequencyprocessor/sieve.go:182:9: undefined: pdata.TimestampFromTime\\n\""}
github.com/go-logr/zapr.(*zapLogger).Error
	/home/runner/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:132
github.com/open-telemetry/opentelemetry-collector-builder/cmd.Execute
	/home/runner/go/pkg/mod/github.com/open-telemetry/opentelemetry-collector-builder@v0.35.0/cmd/root.go:91
main.main
	/home/runner/go/pkg/mod/github.com/open-telemetry/opentelemetry-collector-builder@v0.35.0/main.go:25
runtime.main
	/opt/hostedtoolcache/go/1.17.1/x64/src/runtime/proc.go:255
make[1]: *** [Makefile:49: build] Error 1
make: *** [Makefile:70: otelcol-sumo-linux_arm64] Error 2
Error: Process completed with exit code 2.
```

https://github.com/SumoLogic/sumologic-otel-collector/pull/210/checks?check_run_id=3731559424

and we can take a look at it.

**Review by pmalek-sumo — APPROVED** [review]
Let's merge this in 🚀  Thanks @ppawelecsumo 

I suggest we revisit the comments in code to make it easier to understand for future readers of this processor.

**Pull request merged**